File: test/fsharpTests/IntTest/minimumTest.fs
testCase "minimum(8,18)" 
<| fun _ -> 
    let expected = 8
    Expect.equal expected (Int.minimum 8 18) "error"
testCase "minimum(5,0)" 
<| fun _ -> 
    let expected = 0
    Expect.equal expected (Int.minimum 5 0) "error"
testCase "minimum(-4,-1)" 
<| fun _ -> 
    let expected = -4
    Expect.equal expected (Int.minimum -4 -1) "error"

File: test/ocamlTests/IntTest/minimumTest.ml
test "minimum(8,18)" (fun () -> expect (Int.minimum 8 18) |> toEqual Eq.int 8) ; 
test "minimum(5,0)" (fun () -> expect (Int.minimum 5 0) |> toEqual Eq.int 0) ; 
test "minimum(-4,-1)" (fun () -> expect (Int.minimum -4 -1) |> toEqual Eq.int -4) ; 
